ipatests/test_integration/test_commands.py

```diff
@@ -1170,3 +1170,35 @@ def test_ipa_adtrust_install_with_locale_issue8066(self):
             assert msg2 not in result.stderr_text
         finally:
             bashrc_backup.restore()
+
+    def test_login_wrong_password(self):
+        """Test ipa user login with wrong password
+
+        When ipa user login to machine using wrong password, it
+        should log proper message in /var/log/secure
+
+        related: https://github.com/SSSD/sssd/issues/5139
+        """
+        testuser = 'testuser'
+        tasks.create_active_user(self.master, testuser, 'Secret123')
+
+        # try to login with wrong password
+        sshconn = paramiko.SSHClient()
+        sshconn.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        try:
+            sshconn.connect(self.master.hostname,
+                            username=testuser,
+                            password='WrongPassword')
+        except paramiko.AuthenticationException:
+            pass
+
+        sshconn.close()
+
+        # check if proper message logged
+        exp_msg = ("pam_sss(sshd:auth): received for user {}: 7"
+                   " (Authentication failure)".format(testuser))
+        result = self.master.run_command(['journalctl', '-u', 'sshd'])
```

Comment:
Instead of checking the whole journal, I would add the option `'--since={}'.format(since)` and define
`since = time.strftime('%H:%M:%S')` at the beginning of the test. Otherwise the error may be triggered by another test.